src/main/java/com/fouristhenumber/utilitiesinexcess/common/items/ItemInversionSigilActive.java

```diff
@@ -2,29 +2,551 @@
+        player.addChatMessage(
+            new ChatComponentText(StatCollector.translateToLocal("chat.pseudo_inversion_ritual.header")));
+        if (dimensionOk) {
+            player.addChatMessage(
+                new ChatComponentText(StatCollector.translateToLocal("chat.pseudo_inversion_ritual.correctDimension")));
+        } else if (world.provider.dimensionId == 0) {
+            player.addChatMessage(
+                new ChatComponentText(
+                    StatCollector.translateToLocal("chat.pseudo_inversion_ritual.incorrectDimensionOverworld")));
+        } else if (world.provider.dimensionId == -1) {
+            player.addChatMessage(
+                new ChatComponentText(
+                    StatCollector.translateToLocal("chat.pseudo_inversion_ritual.incorrectDimensionNether")));
+        } else {
+            player.addChatMessage(
+                new ChatComponentText(
+                    StatCollector.translateToLocal("chat.pseudo_inversion_ritual.incorrectDimensionOther")));
+        }
+        player.addChatMessage(
+            new ChatComponentText(
+                StatCollector.translateToLocalFormatted(
+                    "chat.pseudo_inversion_ritual.chestNorthContents",
+                    (chestNorthContentsOk ? "✓" : "✗"))));
+        player.addChatMessage(
+            new ChatComponentText(
+                StatCollector.translateToLocalFormatted(
+                    "chat.pseudo_inversion_ritual.chestEastContents",
+                    (chestEastContentsOk ? "✓" : "✗"))));
+        player.addChatMessage(
+            new ChatComponentText(
+                StatCollector.translateToLocalFormatted(
+                    "chat.pseudo_inversion_ritual.chestSouthContents",
+                    (chestSouthContentsOk ? "✓" : "✗"))));
+        player.addChatMessage(
+            new ChatComponentText(
+                StatCollector.translateToLocalFormatted(
+                    "chat.pseudo_inversion_ritual.chestWestContents",
+                    (chestWestContentsOk ? "✓" : "✗"))));
+        player.addChatMessage(
+            new ChatComponentText(
+                StatCollector
+                    .translateToLocalFormatted("chat.pseudo_inversion_ritual.spiral", (spiralOk ? "✓" : "✗"))));
+        if (dimensionOk && chestNorthContentsOk
+            && chestEastContentsOk
+            && chestSouthContentsOk
+            && chestWestContentsOk
+            && spiralOk) {
+            player.addChatMessage(new ChatComponentTranslation("chat.pseudo_inversion_ritual.ready"));
+        }
+        return true;
+    }
+
+    @SubscribeEvent(priority = EventPriority.NORMAL)
+    public void whenServerTick(TickEvent.ServerTickEvent event) {
+
+        List<EntityPlayer> playerList = getSiegePlayers();
+        for (EntityPlayer player : playerList) {
+            EntitySiegeProperty properties = getProperties(player);
+            if (!properties.siege) {
```

Comment:
this is always false because you just got a list of players with siege = true (see getSiegePlayers())